### PR TITLE
feat: simple circuit experiment

### PR DESCRIPTION
**Pull request by saidabdul80**

### CODE REVIEW

**Comment by Auwalms** [comment]
Closed the PR as https://github.com/cs-dept-ibbul/vlab/pull/56 has this covered.

**Review by Auwalms — APPROVED** [review]

**Review by abdullahikawu — APPROVED** [review]
Looks good!

File: routes/web.php [hunk 1]
```diff
@@ -34,7 +34,16 @@
 
 
 
+ 
+Route::get('/photovoltaic-2','App\Http\Controllers\experimentController@vicelabexp2')->name('vicelab-exp2');
```
Comment (abdullahikawu) [inline routes/web.php:38]: ViceLab?!

File: resources/views/experiment/vicelabexp3.blade.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+       			<!-- experiment  -->
+   			    <div id="mainExp">
+               <v-ribbon></v-ribbon>
+               <v-vicelabexp3 style="display: none;" id="experimentSheet"></v-vicelabexp3>
+               <v-start hourdata="1" munitedata="30" starteddata="{{$started}}" ></v-start>               
+   			    </div>
+   			    <!-- end experiment -->
+
+            <div  class="zero-space vh-2 exprightNav" id="rightNav">               
+               <v-rightnav toolsizes="false" url="{{route('vicelab-exp2')}}" vicelab=1 ></v-rightnav>
```
Comment (abdullahikawu) [inline resources/views/experiment/vicelabexp3.blade.php:43]: ViceLab, kindly rename the word ViceLab across the codebase, instead use VLab, at the least, preferrably use a good description

Comment (abdullahikawu) [reply resources/views/experiment/vicelabexp3.blade.php:43]: same with file names please...

**Pull request merged**